arwen/helpers.go

```diff
@@ -192,8 +193,8 @@ func GetStorageContext(vmHostPtr unsafe.Pointer) StorageContext {
 
 // WithFault returns true if the error is not nil, and uses the remaining gas if the execution has failed
 func WithFault(err error, vmHostPtr unsafe.Pointer, failExecution bool) bool {
-	runtime := GetVMHost(vmHostPtr)
-	return WithFaultAndHost(runtime, err, failExecution)
+	runtimeInstance := GetVMHost(vmHostPtr)
```

Comment:
This should be either `runtimeCtx`, or reverted back to `runtime` given the import alias `goRuntime`.

Reply:
reverted to runtime